CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 ## Next release
 
+## 0.41.0 / 2020-07-29
+
+* [CHANGE] configmap-reload: Update to v0.4.0 (#3334)
+* [CHANGE] Update prometheus compatibility matrix to v2.19.2 (#3316)
+* [FEATURE] Add Synthetic Probes support (#2832)
+* [FEATURE] Support Prometheus vertical compaction (#3281)
+* [ENHANCEMENT] docs: update compatibility warning (#3369)
+* [ENHANCEMENT] Propagate Prometheus image version to statefulset (#3358)
+* [ENHANCEMENT] pkg: instrument resources being tracked by the operator (#3360)
+* [ENHANCEMENT] add SecretListWatchSelector to reduce memory and CPU footprint (#3355)
```

Comment:
nit: let's start all of these with capital letters and make them full sentences.

Reply:
Capitalized all entries. Prefixed changes (i.e. pkg: ...) have been capitalized at the beginning sentence.